src/pages/FeelGoodPage.js

```diff
@@ -20,14 +19,11 @@ export default function FeelGoodPage() {
         />
       </GoBackButton>
       <Header>Scared? Not anymore!</Header>
-
       <CatContainer>
-        {isLoading && <p>Press the button below to see cute cats!</p>}
-        {hasError && isLoading && (
-          <img src={catUrl?.url} alt="" width="300px" />
-        )}
+        <p>Click the button to see cute cats!</p>
+        {hasError && <img src={catResponse?.url} alt="" width="300px" />}
```

Comment:
Do you know if the fetched images have an description? You could use ist as alt text for e.g

Reply:
They don't :( its only the url for the picture and the size